src/App.tsx

```diff
@@ -236,8 +254,73 @@ export default function App({
 		)
 	}
 
+	const startRecording = async () => {
+		try {
+			// Get JWT token from localStorage
+			const jwt = localStorage.getItem(`jwt-${fileId}`)
+			if (!jwt) {
+				throw new Error('No JWT token found')
+			}
+
+			// Construct recording URL with token
+			const boardUrl = window.location.origin + generateUrl(`/apps/whiteboard/recording/${fileId}`)
+				+ `?token=${encodeURIComponent(jwt)}`
+
+			const response = await fetch(`https://nextcloud.local:3002/${fileId}/record`, {
+				method: 'POST',
+				credentials: 'include',
+				headers: {
+					'Content-Type': 'application/json',
+				},
+				body: JSON.stringify({ boardUrl }),
+			})
+
+			if (!response.ok) {
+				throw new Error('Failed to start recording')
+			}
+
+			setIsRecording(true)
+			setRecordingError(null)
+		} catch (error) {
+			console.error('Recording error:', error)
+			setRecordingError('Failed to start recording')
+		}
+	}
+
+	const stopRecording = async () => {
+		try {
+			const response = await fetch(`https://nextcloud.local:3002/${fileId}/stop`, {
```

Comment:
To not forget, we should use the configured whiteboard server URL here, not a hardcoded one